python/cugraph/cugraph/experimental/datasets/dataset.py

```diff
@@ -126,27 +126,37 @@ def get_graph(self, fetch=False, default_direction=True, weights=True):
         fetch : Boolean (default=False)
             Downloads the dataset from the web.
 
-        default_direction : Boolean (defalut=True)
-            Follow the default 'directed' attribute of the dataset. If
-            False, it will return the undirected version of the graph.
+        create_using: cugraph.Graph (instance or class), optional
+        (default=Graph)
+            Specify the type of Graph to create. Can pass in an instance to
+            create a Graph instance with specified 'directed' attribute.
 
-        weights : Boolean (default=True)
-            Include weights.
+        ignore_weights : Boolean (default=False)
+            Weights will be present by default, unless weights are not
+            included in the dataset.
```

Comment:
Minor change to make it slightly more clear.
```suggestion
        ignore_weights : Boolean (default=False)
            Ignores weights in the dataset if True, resulting in an
            unweighted Graph. If False (the default), weights from the
            dataset -if present- will be applied to the Graph. If the
            dataset does not contain weights, the Graph returned will
            be unweighted regardless of ignore_weights.
```